OpenGpxTracker/PreferencesTableViewController.swift

```diff
@@ -29,21 +29,32 @@ let kDefaultsKeyTileServerInt: String = "TileServerInt"
 let kDefaultsKeyUseCache: String = "UseCache"
 
 
-//
-// There are two preferences available:
-//  * use or not cache
-//  * select the map source (tile server)
-//
-// Preferences are kept on UserDefaults with the keys kDefaultKeyTileServerInt (Int)  and kDefaultUseCache (Bool)
-//
+///
+/// There are two preferences available:
+///  * use or not cache
+///  * select the map source (tile server)
+///
+/// Preferences are kept on UserDefaults with the keys `kDefaultKeyTileServerInt` (Int)
+/// and `kDefaultUseCache`` (Bool)
+///
 class PreferencesTableViewController: UITableViewController, UINavigationBarDelegate {
     
+    /// Tile server selected
     var selectedTileServerInt = -1
+    
+    /// Current use of cache
     var currentUseCache: Bool = true
+    
+    /// UserDefaults.standard shortcut
     let defaults = UserDefaults.standard
-    weak var delegate: PreferencesTableViewControllerDelegate?
     
+    /// Delegate for this table view controller.
+    weak var delegate: PreferencesTableViewControllerDelegate?
     
+    /// Does the following:
+    /// 1. Defines the areas for navBar and the Table view
+    /// 2. Sets the title
+    /// 3. Loads the Preferences from defaults (us
```

Comment:
`(us` comment seems incomplete.

Reply:
Fixed in 2511fdfc4f8b59f2a1d462f22d5355a64b67ae9e